uniprot-config/src/test/java/org/uniprot/store/config/searchfield/common/AbstractSearchFieldConfigTest.java

```diff
@@ -87,6 +87,12 @@ void testDoesSearchFieldItemExist() {
         Assertions.assertTrue(testFieldConfig.searchFieldItemExists(fieldName));
     }
 
+    @Test
+    void testDoesSearchFieldItemExist_givenAlias() {
+        String fieldName = "go_alias";
+        Assertions.assertTrue(testFieldConfig.searchFieldItemExists(fieldName));
+    }
```

Comment:
Can you please add a not found test case?

Reply:
its already there

https://github.com/ebi-uniprot/uniprot-store/blob/b331e53e1b62b0da447d92bebff04c9d9544338b/uniprot-config/src/test/java/org/uniprot/store/config/searchfield/common/AbstractSearchFieldConfigTest.java#L97